README.md

```diff
@@ -92,9 +92,41 @@ You can create an account for storing data to AWS S3 buckets.
 
 ```swift
 // In CreateAccountViewController.swift
+
+// fetch the security questions and filter out those that are deprecated
+// JSON returned is structured
```

Comment:
Minor nitpick: would be more readable if this were a full sentence "The returned JSON is structured:`